python/paddle/incubate/autograd/composite_rules.py

```diff
@@ -59,6 +59,13 @@ def composite_batchnorm(
     trainable_statistics,
 ):
     """define composite rule of op batch_norm"""
+    is_amp = False
+    from paddle.fluid.data_feeder import convert_dtype
+
+    if convert_dtype(x.dtype) == "float16":
+        print("Running batch_norm in amp")
```

Comment:
better to add comment to explain we do this

Reply:
sure,but how about in next pr when we support autocast strategy